### PR TITLE
Automate events as well as rotations

### PR DESCRIPTION
**Pull request by evankanderson**
I added two metadata bits for events:

* `title` for the name of the event
* `duration` for how long the event is

Unlike oncall, we show the _next_ event and its duration. You could also use the next method if you want to show who's oncall next in the little cards.

Next up:

1. fetch from URL (from this repo)
2. moving the rotations/events to `knative/community`
3. updating this repo to point to those official locations.

### CODE REVIEW

File: kodata/events/toc-review.rotation [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#@ title: ToC Working Group Update
+#@ duration: 45m
+
+# Start | Group
+2021-03-11T08:30:00-07:00 | Security WG
```
Comment (n3wscott) [inline kodata/events/toc-review.rotation:5]: why is this `-07:00 ` and the others are `:08:00` ? do we need the end date if we have the duration?

Comment (evankanderson) [reply kodata/events/toc-review.rotation]: This is a timezone offset in RFC3339. The others are `17:00:00Z` (Zulu), which is 9AM Pacific in the winter, and 10AM in the summer.

Of course, time zones suck enough that I got this one backwards, and it should be -07:00 next week, and -08:00 now.

Comment (evankanderson) [reply kodata/events/toc-review.rotation]: RFC3339 doesn't support "PST"/"PDT" offsets, but I could be convinced to change the time format...

File: kodata/events/toc-review.rotation [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#@ title: ToC Working Group Update
```
Comment (n3wscott) [inline kodata/events/toc-review.rotation:1]: can you move this to `%root/events` and symlink it back to `kodata/events`? I just did this for the rotations.

Comment (evankanderson) [reply kodata/events/toc-review.rotation]: Done.

**Pull request merged**